spec/acceptance/docker_full_spec.rb

```diff
@@ -0,0 +1,598 @@
+require 'spec_helper_acceptance'
+
+describe 'docker full acceptance tests section 1' do
+  context 'test 1.1 - Install and configure Docker and Docker Daemon' do
```

Comment:
I'm not sure I like the `test 1.1` portion, I don't think keeping references like this will be particularly sustainable.